Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/CompositeSequences/EconomizerMultiZone.mo

```diff
@@ -0,0 +1,179 @@
+    "Outdoor air (OA) temperature" annotation (Placement(transformation(extent={{-140,130},{-120,150}}),
+    iconTransformation(extent={{-120,110},{-100,130}})));
+  CDL.Interfaces.RealInput TOutCut(unit="K", quantity = "ThermodynamicTemperature")
+    "OA temperature high limit cutoff. For differential dry bulb temeprature condition use return air temperature measurement"
+    annotation (Placement(transformation(extent={{-140,110},{-120,130}}),
+        iconTransformation(extent={{-120,90},{-100,110}})));
+  CDL.Interfaces.RealInput hOut(unit="J/kg", quantity="SpecificEnergy") if fixEnt
+    "Outdoor air enthalpy" annotation (Placement(transformation(extent={{-140,90},{-120,110}}),
+    iconTransformation(extent={{-120,70},{-100,90}})));
+  CDL.Interfaces.RealInput hOutCut(unit="J/kg", quantity="SpecificEnergy") if fixEnt
+    "OA enthalpy high limit cutoff. For differential enthalpy use return air enthalpy measurement"
+    annotation (Placement(transformation(extent={{-140,70},{-120,90}}),
+        iconTransformation(extent={{-120,50},{-100,70}})));
+  CDL.Interfaces.RealInput VOut(unit="m3/s")
+    "Measured outdoor volumetirc airflow rate [fixme: which quantity attribute should we use? add for all V]"
+    annotation (Placement(transformation(extent={{-140,10},{-120,30}}),
+        iconTransformation(extent={{-120,-10},{-100,10}})));
+  CDL.Interfaces.RealInput VOutMinSet(unit="m3/s")
+    "Minimum outdoor volumetric airflow rate setpoint"
+    annotation (Placement(transformation(extent={{-140,-10},{-120,10}}),
+        iconTransformation(extent={{-120,-30},{-100,-10}})));
+  CDL.Interfaces.IntegerInput uFreProSta "Freeze protection status"
+    annotation (Placement(transformation(extent={{-140,-130},{-120,-110}}),
+      iconTransformation(extent={{-120,-110},{-100,-90}})));
+  CDL.Interfaces.IntegerInput uOperationMode "AHU operation mode status signal"
+    annotation (Placement(transformation(extent={{-140,-90},{-120,-70}}),
+      iconTransformation(extent={{-120,-70},{-100,-50}})));
+  CDL.Interfaces.IntegerInput uZoneState "Zone state input"
+    annotation (Placement(transformation(extent={{-140,-110},{-120,-90}}),
+    iconTransformation(extent={{-120,-90},{-100,-70}})));
+  CDL.Interfaces.BooleanInput uSupFan "Supply fan status"
+    annotation (Placement(transformation(extent={{-140,-50},{-120,-30}}),
+        iconTransformation(extent={{-120,-50},{-100,-30}})));
+
+  CDL.Interfaces.RealOutput yRetDamPos "Return air damper position"
+    annotation (Placement(transformation(extent={{120,30},{140,50}}),
+    iconTransformation(extent={{100,10}, {120,30}})));
+  CDL.Interfaces.RealOutput yOutDamPos "Outdoor air damper position"
+    annotation (Placement(transformation(extent={{120,-50},{140,-30}}),
+    iconTransformation(extent={{100,-30}, {120,-10}})));
+
+  AtomicSequences.EconEnableDisableMultiZone econEnableDisableMultiZone(
+    delEntHis=delEntHis,
```

Comment:
Validation tests are looking at some of the outputs of these blocks, which is why I did not protect them. Let me know if there is a suitable workaround.